Runtime/Scripts/MelMessaging.cs

```diff
@@ -0,0 +1,112 @@
+﻿/**
```

Comment:
Great! We need to manually update mel.aar binary as needed in the GitHub so developers pull from live repo with the binary.

Reply:
No need to manually added it, I added mel.aar(1.0.4) and their is functionality in the EditorScript to move it from the package to Plugins/MobiledgeX/Android/mel.aar